### ISSUE TITLE
Does not show window in Ubuntu 18.04.1

### ISSUE THREAD

**Issue by VTVL** [yktoo/indicator-sound-switcher#64]
Thanks for developing this tool.
After upgrading from Ubuntu MATE 16.04.5  to 18.04.1   indicator-sound-switcher  runs but the window that shows the sound cards does not display.  After removing and doing a fresh PPA setup and install the problem persists.
the -vv output is attached
[indicator-sound-switcher-vv.txt](https://github.com/yktoo/indicator-sound-switcher/files/2290393/indicator-sound-switcher-vv.txt)

**Comment by yktoo** [comment]
What do you mean with "the window that shows the sound cards", is it the indicator icon? If that's the case and the indicator is not showing, you probably need to use [this solution](https://github.com/yktoo/indicator-sound-switcher/issues/62#issuecomment-372107128) from #62.

**Comment by VTVL** [comment]
I mean this window
![example-window](https://user-images.githubusercontent.com/21249637/44159021-8de6b800-a0b6-11e8-902c-3becee2fce53.png)
This is just the example on your web page.
The icon is present in the menu and panel. When clicked it starts but there is no window.

**Comment by yktoo** [comment]
Could you attach an actual screenshot of what you're seeing?

**Comment by VTVL** [comment]
The desktop is a standard Ubuntu MATE 18.04.  I have the icon for indicator-sound-switcher on the top panel as in your example.   The command is indicator-sound-switcher.   If I start it via a terminal with the -VV option the output is as shown in the file attached to my first message.  The window does not appear.  The system monitor shows that the indicator-sound-switcher process is sleeping.

**Comment by yktoo** [comment]
Sounds like the desktop environment's problem to me because it handles the menu. And I can't see anything irregular in the output either.

**Comment by VTVL** [comment]
Solved! Sort of...
I just discovered something new in the Add To Panel menu of apps. Actually it is not new; just never used.  I added Indicator Applet Complete to the Panel.  Then I explored what it was about.  Some icons had moved.  I saw that there were now two indicator-sound-switcher icons.   I clicked the newer icon and the window appeared.   Then I deleted to original icon.
Thanks for your prompt replies.
